client/client_test.go

```diff
@@ -1243,6 +1243,59 @@ func TestClient_UpdateNodeFromDevicesAccumulates(t *testing.T) {
 
 }
 
+func TestClient_UpdateNodeFromFingerprintKeepsConfig(t *testing.T) {
```

Comment:
We've started adding comments to tests to try to summarize their core assertion to make it easy to understand at a glance.

```suggestion
// TestClient_UpdateNodeFromFingerprintKeepsConfig asserts manually configured
// network interfaces take precedence over fingerprinted ones.
func TestClient_UpdateNodeFromFingerprintKeepsConfig(t *testing.T) {
```